xml/SectorCategory.xml

```diff
@@ -163,7 +163,7 @@
                 <narrative xml:lang="fr"/>
             </description>
         </codelist-item>
-        <codelist-item status="withdrawn">
+        <codelist-item status="withdrawn" withdrawal-date="2020-02-28">
```

Comment:
This code has been withdrawn since at least June 2017.

Reply:
Thanks! Checking with the OECD and will be updating the dates once we hear back from them.